deepinv/physics/forward.py

```diff
@@ -346,6 +362,20 @@ def A_adjoint(self, y):
 
         return self.A_adj(y)
 
+    def A_jvp(self, x, v):
+        r""" "
+        Applies the Jacobian of the forward operator to the input x and perturbation v, defined as:
```

Comment:
Computes the product between the Jacobian of the forward operator :math:A evaluated at :math:x and the vector :math:v, defined as: